Changelog.md

```diff
@@ -1,6 +1,8 @@
-## Gazebo GUI 7
+## Gazebo GUI 8
+
+### Gazebo GUI 8.0.0 (202X-XX-XX)
```

Comment:
```suggestion
### Gazebo GUI 8.X.X (202X-XX-XX)
```

Reply:
Ahh those are ok, because the first release will be 0.0